zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -223,13 +223,6 @@ paths:
             schema:
               $ref: '#/components/schemas/ProcessInstanceSearchQueryRequest'
       responses:
-        '200':
-          description: >
-            The Process Instance Search successful response.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/ProcessInstanceSearchQueryResponse"
```

Comment:
❌ This needs to stay. You rather need to remove the `ProcessInstanceSearchQueryResponse` you introduced in the `responses` with this PR.